cmd/lifecycle/detector.go

```diff
@@ -119,13 +174,43 @@ func (da detectArgs) verifyBuildpackApis(order lifecycle.BuildpackOrder) error {
 	return nil
 }
 
-func (d *detectCmd) writeData(group lifecycle.BuildpackGroup, plan lifecycle.BuildPlan) error {
-	if err := lifecycle.WriteTOML(d.groupPath, group); err != nil {
+func (da detectArgs) validateBuildpacks(order lifecycle.BuildpackOrder) error {
```

Comment:
It might be nice for this to have a more specific name, perhaps `verifyBuildpackPrivileges` ...or something. `validateBuildpacks` is very vague :)